spec/models/database_spec.rb

```diff
@@ -0,0 +1,97 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+RSpec.describe "Database" do
+  excluded_models = ["Gateway", "PayPalExpress", "Bogus", "BogusSimple"].freeze
```

Comment:
Why exactly are these excluded? Is there a logic we can find to remove this list?

Reply:
They use `delegate_belongs_to`, which is deprecated and [has actually been removed from Spree](https://github.com/spree/spree/pull/8275).

It took me awhile, but I managed to remove it completely from the codebase since the native rails `delegate` replaces it. #11579